docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -196,6 +196,43 @@ args = ["-vv", "--no-header"]
 
 Then, in your CI script or config, set the environment variable `PANTS_CONFIG_FILES=pants.ci.toml` to use this new config file, in addition to `pants.toml`.
 
+Using with pre-commit
+---------------------
+
+To allow developers to check code before committing or pushing, you can supply a [pre-commit](https://pre-commit.com/) hook that will run Pants commands before.  This can help have a faster feedback loop and reduce load and cost on your CI systems.
+
+However, using `--changed-since=origin/main` is not as useful locally.  Developers my have unstaged or untracked files lying around in their working directory, or their Git references may be out of date or named differently than the ones you use in CI.  Instead, it's better to leverage pre-commit's ability to pass changed filenames to commands to Pants, and tell Pants to ignore files it's been told not to track:
```

Comment:
```suggestion
However, using `--changed-since=origin/main` is not as useful locally.  Developers my have unstaged or untracked files lying around in their working directory, or their Git references may be out of date or named differently than the ones you use in CI.  Instead, it's often better to leverage pre-commit's ability to pass changed filenames to commands to Pants, and tell Pants to ignore files it's been told not to track:
```